infra/src/main/java/com/redhat/service/smartevents/infra/models/dto/BridgeDTO.java

```diff
@@ -18,6 +18,9 @@ public class BridgeDTO {
     @JsonProperty("customerId")
     private String customerId;
 
+    @JsonProperty("user_name")
+    private String userName;
```

Comment:
I think it would be better to align with `kafka`: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/0af018cb9949bfdde439839974f08f330a210c9f/internal/kafka/internal/handlers/kafka.go#L58. 

They use `owner` instead of `userName`. If you agree with that, I'd replace it in all the classes. wdyt?

Reply:
OK, I'll do that.. it's also referred to as `owner` [here](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/internal/connector/internal/services/connector_cluster.go#L145).

Reply:
@r00ta Done.